utils/synth_wav.sh

```diff
@@ -127,8 +136,12 @@ function download_vocoder_models () {
     case "${vocoder_models}" in
         "ljspeech.wavenet.softmax.ns.v1") share_url="https://drive.google.com/open?id=1eA1VcRS9jzFa-DovyTgJLQ_jmwOLIi8L";;
         "ljspeech.wavenet.mol.v1") share_url="https://drive.google.com/open?id=1sY7gEUg39QaO1szuN62-Llst9TrFno2t";;
```

Comment:
Did you forget to add`ljspeech.wavenet.mol.v2` perhaps?

Reply:
Here I decided to list only the no limit ljspeech models not to be confused.
Instead of here, I made the table of all of the pretrained models in `README.md`. How do you think?